manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

```
@@ -0,0 +1,96 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: VideoLAN.VLC
+PackageVersion: 3.0.18
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- vlc
+FileExtensions:
+- 264
+- 3ga
+- 3gp
+- aac
+- avi
+- cda
+- dash
+- dvr
+- flac
+- ifo
+- m2t
+- m2ts
+- m3u8
+- m4v
+- mkv
+- mov
+- mp3
+- mp4
+- mpg
+- mts
+- ogg
+- ogv
+- opus
+- pls
+- rec
+- rmvb
+- snd
+- sub
+- ts
+- vob
+- webm
+- wma
+- wmv
+- zab
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi

```

```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
```
Please use the direct URL to the installer.